src/utils/provideLocalStorageDataClass.ts

```diff
@@ -91,3 +91,33 @@ function isStringObject(item: unknown): item is Record<string, unknown> {
   if (!isObject(item)) return false
   return Object.keys(item).every((key) => typeof key === 'string')
 }
+
+function sortItems(
+  items: [string, unknown][],
+  order: Array<[string, 'asc' | 'desc']>,
+): [string, unknown][] {
+  if (order.length === 0) return items
+
+  return [...items].sort((a, b) => {
```

Comment:
* [ ] Since we're using lodash anyway, I wonder if we could just use `orderBy` instead of rolling our own? It accepts nested field paths, direction, and multiple criteria.